### PR TITLE
Update changelog for closed PR #774

### PR DESCRIPTION
**Pull request by richardkoehler**
PR Description
--------------

Update changelog for PR #774 which has already been closed. Re-opening PR #774 was not possible
@hoechenberger 

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [x] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [x] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -25,7 +25,12 @@ Authors
 
 * `Alex Rockhill`_
 * `Richard Höchenberger`_
+* `Stefan Appelhoff`_
+* `Diego Lozano-Soldevilla`_
+* `Eduard Ort`_
+* `Alexandre Gramfort`_
```
Comment (richardkoehler) [inline doc/whats_new.rst:31]: Not sure what happened here, but I didn't actually add any authors other than me

**Comment by hoechenberger** [comment]
I suppose you were working on the wrong branch or something … I'll see if I can fix it!

**Comment by richardkoehler** [comment]
Actually, I had already deleted the branch.. 🙄

**Comment by hoechenberger** [comment]
> Actually, I had already deleted the branch.. 🙄

Yes but that shouldn't matter, something went wrong elsewhere … as if you added the changelog entry to our 0.7 maintenance branch or something. Anyway, I believe I managed to fix it

**Comment by hoechenberger** [comment]
Thanks, @richardkoehler!

**Pull request merged**